handlers/main.yml

```diff
@@ -11,3 +11,7 @@
   sudo: yes
   command: timedatectl set-ntp true
   notify: Reload systemd-timesyncd
+
+- name: Run logrotate
+  become: yes
+  command: /usr/sbin/logrotate -fv /etc/logrotate.d/docker-container-logrotate
```

Comment:
Yes, its just a validation.
I've spent quite sometime trying to add some cleanup job to remove the added configuration and it seems its not a as easy as it sounds. Ansible provides `block-rescue` option for `tasks`, but thats still not available in `handlers` https://github.com/ansible/ansible/issues/14270. I tried some other ugly ways using register as well but it isn't fruitful. I think I should not spend much time on this. 
As for the failures, the only failures would be if `logrotated` service is unavailable or if you copy a file on the node and then refer it by different name in the handler. In both cases, the file would be copied over to the host and whenever the `logrotated` service is triggered to run(assuming it runs correctly) it should run the rotation. For second case, its an ansible user error and we should see the handler task failure on error logs and one should fix that. All said, the gist of this story is, copying the logrotate config file is not going to render the host unhealthy. So we should be okay there. I'll drop this request at this point.